aiidalab_widgets_base/process.py

```diff
@@ -412,8 +412,12 @@ def update(self):
 
         output_file_path = None
         if 'remote_folder' in self.calculation.outputs:
-            output_file_path = os.path.join(self.calculation.outputs.remote_folder.get_remote_path(),
-                                            self.calculation.attributes['output_filename'])
+            try:
+                output_file_path = os.path.join(self.calculation.outputs.remote_folder.get_remote_path(),
+                                                self.calculation.attributes['output_filename'])
+            except KeyError:
+                self.placeholder = f"The `output_filename` attribute is not set for {self.calculation.process_class}. " \
+                "Nothing to show."
         if output_file_path and os.path.exists(output_file_path):
```

Comment:
In relation to my previous comment on line 414, I believe the control flow is incorrect here. Is the `output_file_path` variable even set at this point if we caught a `KeyError` earlier?